CONTRIBUTING.md

```diff
@@ -123,6 +123,35 @@ You can either specify it in the YAML header (`thumbnail: image.webp`) or add `{
 ![Slope map](slope.webp){.preview-image}
 ```
 
+## External tutorials
+
+If you have your own website or blog where you regularly post your GRASS courses, workshops or tutorials, 
+you can have those listed as external tutorials. If your website is fully dedicated to GRASS tutorials, 
```

Comment:
I would probably leave that option out, having them as individual tutorials is better and makes the instructions simpler.